Core/src/RestTrait.php

```diff
@@ -104,4 +104,28 @@ public function send($resource, $method, array $options = [], $whitelisted = fal
             throw $e;
         }
     }
+
+    /**
+     * Return a custom API endpoint in the proper format, or default if none provided.
+     *
+     * @param string $default
+     * @param array $config
+     * @return string
+     */
+    private function getApiEndpoint($default, array $config)
+    {
+        $res = isset($config['apiEndpoint']) && $config['apiEndpoint']
```

Comment:
If the isset check passes, what is the second half of the statement checking for?